pkgs/os-specific/linux/cpuset/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv
+, fetchgit
+, python3Packages
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "cpuset";
+  version = "1.6pre1";
+  name = "${pname}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.